appOPHD/UI/FileIo.cpp

```diff
@@ -103,6 +103,16 @@ void FileIo::setMode(FileOperation fileOp)
 }
 
 
+void FileIo::showOpen(const std::string& directory)
+{
+	this->scanDirectory(directory);
+	mMode = FileOperation::Load;
+	title(constants::WindowFileIoTitleLoad);
+	mFileOperation.text(constants::WindowFileIoLoad);
+	this->show();
```

Comment:
Huh, this didn't stand out to me before, though the two `this->` parts should be unnecessary. Sometimes they're needed in a template class, though this isn't a template class.